async/src/test/java/org/odk/collect/async/WorkerAdapterTest.kt

```diff
@@ -69,6 +75,63 @@ class WorkerAdapterTest {
         assertThat(workerAdapter.doWork(), `is`(ListenableWorker.Result.failure()))
     }
 
+    @Test
+    fun `when numberOfRetries is not specified should data contain LAST_UNIQUE_ATTEMPT equal true`() {
+        whenever(workerAdapter.runAttemptCount).thenReturn(1)
+        whenever(spec.numberOfRetries).thenReturn(-1)
+        whenever(spec.getTask(any(), any())).thenReturn(Supplier { true })
+
+        workerAdapter.doWork()
+
+        verify(spec).getTask(
+            any(),
+            eq(
+                mapOf(
+                    TaskSpec.DATA_PROJECT_ID to "123",
+                    TaskSpec.LAST_UNIQUE_ATTEMPT to "true"
+                )
+            )
+        )
+    }
+
+    @Test
+    fun `when numberOfRetries is specified and it is higher than runAttemptCount, should data contain LAST_UNIQUE_ATTEMPT equal false`() {
```

Comment:
Again these tests don't need `should` in them really. Could just be `..runAttemptCount, data contains...`

Reply:
Fixed.